simulator/infra/cache/cache_tag_array.h

```diff
@@ -96,6 +96,8 @@ class CacheTagArray : public CacheTagArraySize
             uint32 addr_size_in_bits = 32
         );
 
+        // hit or not
+        bool is_hit( Addr addr) { return read( addr).first; }
```

Comment:
That method has a confusing name, it seems that it does not modify the cache _(but it does a lot!)_ . Could you please rename it to `lookup`?